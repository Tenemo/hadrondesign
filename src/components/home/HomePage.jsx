import { Link } from 'react-router-dom';

class HomePage extends React.Component {
    render() {
        return pug`
            .jumbotron
                h1 Pluralsight Administration
                p React, Redux and React Router in ES6 for ultra-responsive web apps.
                Link(to="about").btn.btn-primary.btn-lg Learn more
    `;
    }
}

export default HomePage;
