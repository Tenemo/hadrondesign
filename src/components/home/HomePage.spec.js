import React from 'react';
import HomePage from './HomePage';

describe('HomePage', () => {
    // it('should have an h1 About header', () => {
    // });
    it('matches snapshot', () => {
        const wrapper = shallow(pug`
            HomePage
        `);
        expect(wrapper).toMatchSnapshot();
    });
});