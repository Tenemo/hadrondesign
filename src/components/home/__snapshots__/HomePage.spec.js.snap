// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage matches snapshot 1`] = `
<div
  className="jumbotron"
>
  <h1>
    Pluralsight Administration
  </h1>
  <p>
    React, Redux and React Router in ES6 for ultra-responsive web apps.
  </p>
  <Link
    className="btn btn-primary btn-lg"
    replace={false}
    to="about"
  >
    Learn more
  </Link>
</div>
`;
